app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<h1 class='text-center mt-5'>Recipes</h1>
+
+<style>
+  .recipe-item {
+    display: flex;
+    flex-direction: column; 
+    gap: 10px;
+    padding: 10px; 
+  }
+  
+  .name-and-button {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  .description-container {
+    word-wrap: break-word; 
+    text-align: left; 
+  }
+  
+  .bold-name {
+    font-weight: bold; 
+  }
+</style>
+
+<div class="container mt-5">
+  <div class="text-right mb-3 d-flex justify-content-end align-content-center">
+    <%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-success" %>
+  </div>
+
+  <% if @recipes.any? %>
+    <div class="card">
+      <div class="card-body">
+        <% @recipes.each do |recipe| %>
+          <div class="recipe-item">
+            <div class="name-and-button">
+              <span class="bold-name"><%= recipe.name %></span>
+              <%= button_to "Remove", recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger btn-sm" %>
+            </div>
+            <div class="description-container">
+              <%= recipe.description %>
+            </div>
+          </div>
+          <hr> 
+        <% end %>
+      </div>
+    </div>
+  <% else %>
+    <div class="text-center" data-testid="no-recipes-message">
+      <p>No recipes found.</p>
+    </div>
```

Comment:
- [ ] @Team Great work implementing the change requested, however as the previous reviewer noted your recipe page layout doesn't match the wireframe layout given, which is part of the project requirement
![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/49d071ab-72d9-4c90-a088-2c4368252ce9)


Wireframe Layout          |  Your Layout 
:-------------------------:|:-------------------------:
![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/354fc17d-6b4c-4c45-b9c2-89bfb07edb37)  | ![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/20410bd5-35b2-47bb-bb1a-5f6441ef4925)

Reply:
@danmainah Dear Sir, index.html.erb is not like that. Please check again. 
<h1 class='text-center mt-5'>Recipes</h1>

<style>
  .recipe-item {
    display: flex;
    flex-direction: column; 
    gap: 10px;
    padding: 10px; 
  }
  
  .name-and-button {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
  
  .description-container {
    word-wrap: break-word; 
    text-align: left; 
  }
  
  .bold-name {
    font-weight: bold; 
  }
</style>

<div class="container mt-5">
  <div class="text-right mb-3 d-flex justify-content-end align-content-center">
    <%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-success" %>
  </div>

  <% if @recipes.any? %>
    <div class="card">
      <div class="card-body">
        <% @recipes.each do |recipe| %>
          <div class="recipe-item">
            <div class="name-and-button">
              <span class="bold-name"><%= recipe.name %></span>
              <%= button_to "Remove", recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger btn-sm" %>
            </div>
            <div class="description-container">
              <%= recipe.description %>
            </div>
          </div>
          <hr> 
        <% end %>
      </div>
    </div>
  <% else %>
    <div class="text-center" data-testid="no-recipes-message">
      <p>No recipes found.</p>
    </div>
  <% end %>
</div>

Reply:
> @danmainah Dear Sir, index.html.erb is not like that. Please check again.
> 
> # Recipes
> <style> .recipe-item { display: flex; flex-direction: column; gap: 10px; padding: 10px; } .name-and-button { display: flex; justify-content: space-between; align-items: center; } .description-container { word-wrap: break-word; text-align: left; } .bold-name { font-weight: bold; } </style>
> <%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-success" %>
> <% if @recipes.any? %>
> 
> 
> 
> <% @recipes.each do |recipe| %>
> 
> 
> <%= recipe.name %>
> <%= button_to "Remove", recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger btn-sm" %>
> 
> 
> <%= recipe.description %>
> 
> 
> 
> <% end %>
> 
> 
> <% else %>
> 
> No recipes found.
> 
> 
> 
> <% end %>
@mryeminthu this is the code I used from the dev branch 
![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/e1f83918-bf3e-49ea-aca7-90bcac6c0b12)